changelog-endpoint/README.md

```diff
@@ -0,0 +1,36 @@
+# Changelog endpoint
+The `changelog-endpoint` module is a REST service intended for Iroha data model migrations: adding signatories, setting account details, creating domains, appending roles etc.
+
+## Simplified flow
+1) The service executes given Groovy script that implements `ChangelogInterface`.
+2) Created changelog transaction(generated by Groovy script) is accompanied with a changelog history transaction in order to keep track of changes.
```

Comment:
Changelog transaction (generated by Groovy script) created is accompanied by a changelog history transaction in order to keep track of changes.